Star Trek Adventurers 2d20/Star Trek Adventures 2d20.css

```diff
@@ -0,0 +1,162 @@
+/* === GENERAL === */
+.charsheet {
+    min-width: 820px;
+    background: #FFFFFF;
+    background-image: url("https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/Star Trek Adventurers 2d20/Sheet_blank.PNG");
```

Comment:
I'm just a contrib like you and I put my static images on imgur just because that's what the official roll20 5e sheet did. I just mention it here because it made it hard to try this out. In the end, when this gets merged it shouldn't be a problem unless the images change.